src/App.js

```diff
@@ -2,17 +2,17 @@ import './App.css';
 import initialCards from './cards-data';
 import Player from './components/Player';
 import CardList from './components/CardList';
-import { useState } from 'react';
+import { useContext, useState } from 'react';
 import ExecutePassButton from './components/ExecutePassButton';
+import { GameContext } from './Context/GameContext.js';
 
 function App() {
   const [deck, setDeck] = useState(initialCards);
   const [playerOneHand, setPlayerOneHand] = useState([]);
-  const [selectedCard, setSelectedCard] = useState();
   const [playerTwoHand, setPlayerTwoHand] = useState([]);
   const [playerThreeHand, setPlayerThreeHand] = useState([]);
-  const [from, setFrom] = useState('deck');
-  const [to, setTo] = useState(1);
+
+  const { selectedCard, setSelectedCard, from, setFrom, to, setTo } = useContext(GameContext);
 
   function findCardIndex(value, suit, cards) {
```

Comment:
Notice how these functions are not being used here? The only reason they were living in `app` instead of `ExecutePassFunction` was because of state limitations that can be fixed by putting the rest of this state into context. Not docking any points for this, but try to see if you can move them out of here and into where they are being used. One less prop to pass, a cleaner-looking `app.js`.

Reply:
Yes! I think I just missed doing this step! I'll take care of this when I can during our hack-day tomorrow!